<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LuisGabriel" failures="1" tests="1" name="VerifyCheckoutButtonDisabledWhenCarEmpty" time="23.326" errors="0" timestamp="2021-01-03T12:17:07 CST">
  <testcase classname="finaltc.VerifyCheckoutButtonDisabledWhenCarEmpty" name="verifyCheckoutButtonDisabledWhenCarEmpty" time="10.112">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at finaltc.VerifyCheckoutButtonDisabledWhenCarEmpty.verifyCheckoutButtonDisabledWhenCarEmpty(VerifyCheckoutButtonDisabledWhenCarEmpty.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- verifyCheckoutButtonDisabledWhenCarEmpty -->
</testsuite> <!-- VerifyCheckoutButtonDisabledWhenCarEmpty -->
